components/climber.py

```diff
@@ -1,28 +1,26 @@
+import math
+
 from magicbot import feedback, tunable
-from rev import LimitSwitchConfig, SparkMax, SparkMaxConfig
+from rev import SparkMax, SparkMaxConfig
+from wpilib import DutyCycleEncoder
 
-from ids import SparkId
+from ids import DioChannel, SparkId
 
 
 class ClimberComponent:
     target_speed = 0.0
     winch_voltage = tunable(12.0)
+    MIN_ANGLE = 40
+    MAX_ANGLE = 90
 
     def __init__(self) -> None:
+        self.encoder = DutyCycleEncoder(DioChannel.CLIMBER_ENCODER, math.tau, 0)
```

Comment:
Why do we convert [0,1] to radians, to then _manually_ convert everything to degrees?